### PR TITLE
iOS doc update

### PR DESCRIPTION
**Pull request by nqkhang**
resolve https://github.com/streamroot/tech-board/issues/6486

Remove Carthage from the doc, add Limitation section for Mesh 

@gusai-streamroot and @sankyboi , could you please have a look?

### CODE REVIEW

File: mesh-delivery/ios/AVPlayerMeshDelivery/README.md [hunk 1]
```diff
@@ -1,19 +1,24 @@
 # Mesh Delivery SDK Integration for iOS, iPadOS and tvOS
 
 ## Prerequisite
+
```
Comment (sankyboi) [inline mesh-delivery/ios/AVPlayerMeshDelivery/README.md:4]: We should either not leave an empty line after headers everywhere, or leave them everywhere.  Idk how to see the pages without deploying to prod, so I'll let  you decide whichever is better :D

**Review by sankyboi — COMMENTED** [review]
Only the feedback on the empty lines after headers - Either add everywhere or remove everywhere, but pages should be uniform in that respect. Otherwise, LGTM :D

**Comment by gusai-streamroot** [comment]
Both comments by Sanket look pertinent to me

File: mesh-delivery/ios/AVPlayerMeshDelivery/README.md (hunk 1 above)
Comment (nqkhang) [reply mesh-delivery/ios/AVPlayerMeshDelivery/README.md:4]: I have a plugin in VSCode , which format the code once I press Ctrl+S/ Cmd+S.
For now I will "undo" this kind of format "just to be safe"

**Comment by nqkhang** [comment]
Updated, they should be ok now. Quick "final" check, @sankyboi ?

**Review by sankyboi — COMMENTED** [review]
Looks good!

**Pull request merged**